builtin/for-each-repo.c

```diff
@@ -32,13 +32,16 @@ static int run_command_on_repo(const char *path, int argc, const char ** argv)
 int cmd_for_each_repo(int argc, const char **argv, const char *prefix)
 {
 	static const char *config_key = NULL;
+	int keep_going = 0;
 	int i, result = 0;
 	const struct string_list *values;
 	int err;
 
 	const struct option options[] = {
 		OPT_STRING(0, "config", &config_key, N_("config"),
 			   N_("config key storing a list of repository paths")),
+		OPT_BOOL(0, "keep-going", &keep_going,
```

Comment:
You should also update for_each_repo_usage accordingly.

Reply:
Since there is a test case that verifies that the usage and the synopsis match for any given command, this would be a required change if the synopsis change, and could not be changed if the synopsis is left unchanged.